script/build.fsx

```diff
@@ -72,11 +72,17 @@ Target "Package" (fun _ ->
     ensureDirectory artifactsNuGetDir
     ensureDirectory artifactsBuildDir
     
+    let nuspecProps = getNuspecProperties(File.ReadAllText(nuspecPath))
+    
+    let version = match TeamCityBuildNumber.IsSome with
+                      | true ->  sprintf "%s%s-apx" nuspecProps.Version TeamCityBuildNumber.Value
+                      | false -> sprintf "%s-apx" nuspecProps.Version
```

Comment:
Here, you could actually do the following

```F#
let version = match TeamCityBuildNumber with
                    | Some(num) -> sprintf "%s%s-apx" nuspecProps.Version num
                    | None -> sprintf "%s-apx" nuspecProps.Version
```

There's probably also a way to just omit a value if the option is None within `sprintf`.